### PR TITLE
Add unminified build

### PR DESCRIPTION
**Pull request by jochenberger**
This adds another webpack configuration for an unminified production build. This PR supersedes #273.

### CODE REVIEW

File: webpack.production.min.config.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+var webpack = require('webpack');
+
+module.exports = {
+  devtool: '#eval-source-map',
```
Comment (jochenberger) [inline webpack.production.min.config.js]: This still uses `eval`, which should probably be avoided in production (see #217), maybe this shhould go into another build, maybe `.debug.js`.

**Comment by AllenFang** [comment]
@jochenberger, I think this PR will generate two result file in ```dist``` folder, one in minified and the other is non. but for the user who install by npm, they always got the minified one. Because  the following line defined in ```package.json```:
```"main": "./dist/react-bootstrap-table.min.js",```

Which means you used ```react-bootstrap-table``` by imported scripts to HTML instead of npm, am I right?

**Comment by jochenberger** [comment]
Yes, almost, I use it as an AMD module via Require.js. I can control which file to use in my app's configuration.

**Comment by AllenFang** [comment]
@jochenberger, thanks your contribution, this PR is merged in v1.5.2

**Pull request merged**